bedrock/firefox/templates/firefox/faq.html

```diff
@@ -0,0 +1,105 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% from "macros-protocol.html" import hero with context %}
+
+{% block page_title %}{{ ftl('page-title') }}{% endblock %}
+{% block page_desc %} {{ ftl('page-desc') }} {% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-faq') }}
+{% endblock %}
+
+{% block content %}
+<div class="mzp-l-content">
+
+  {% call hero(
+    title=ftl('firefox-faq'),
+    class='mzp-has-image mzp-t-firefox mzp-t-product-firefox',
+    image_url='img/firefox/browsers/quantum/browser.jpg',
+    include_highres_image=True,
+    include_cta=True,
+    heading_level=1
+  ) %}
```

Comment:
Thanks Maxx! This comment will be a WONTFIX from me, because the wordmark logo is not the convention we use for [page heros](https://protocol.mozilla.org/patterns/organisms/hero.html#a-firefox-branded-hero-with-image).